### ISSUE TITLE
Why rpn_class_loss use K.sparse_categorical_crossentropy ?

### ISSUE THREAD

**Issue by 5730279821-TA** [matterport/Mask_RCNN#1107]
rpn_class_loss is a loss from RPN classifier logits for FG/BG. 
Why this loss use sparse_categorical_crossentropy ?
Should it be binary_crossentropy ? because it only 2 classes.

**Comment by ApoorvaSuresh** [comment]
Hi, I was wondering the same thing!
Did you get to know why? or Did you get a chance to use binary Crossentropy for rpn classification?
